services/ory-svc/main.go

```diff
@@ -86,14 +103,59 @@ func afterRegistrationWebhookHandler(ctx context.Context, in *daprcmn.Invocation
 		return nil, newErrAndLog(ctx, err.Error())
 	}
 
-	userRegisteredEvent := &events.UserRegisteredEvent{
+	ctx, cancel, err := prepCtxForSvcToSvcCall(ctx, "user-svc")
+	if err != nil {
+		return nil, newErrAndLog(ctx, err.Error())
+	}
+	defer cancel()
+
+	daprClient := common.MustNewDaprGRPCClient()
+	userSvc := userSvcPb.NewUserServiceClient(daprClient.GrpcClientConn())
+	organizationSvc := userSvcPb.NewOrganizationServiceClient(daprClient.GrpcClientConn())
+
+	_, err = userSvc.CreateUser(ctx, &userSvcPb.CreateUserRequest{
 		Id:       userID.String(),
 		Email:    payload.Email,
 		Nickname: payload.Nickname,
 		Name:     payload.Name,
+	})
+
+	if err != nil {
+		return nil, newErrAndLog(ctx, err.Error())
+	}
+
+	trueVal := true
```

Comment:
Really? Maybe we should create a helper function for passing a boolean pointer based on a fixed value.